docs/README.md

```diff
@@ -3,7 +3,7 @@
 You can run the following steps to generate the documentation website:
 
 ```
-tox -e docbuild, doclint
+tox -e docbuild,doclinting
```

Comment:
Having a space there was a bug, causing only docbuild to be run.

```
tox -e docbuild,doclint
```
->
```
ERROR: unknown environment 'doclint'
```
So also fixed that -> doclinting.